File: Code/Components/DanpheEMR.ServerModel/ClinicalModels/EmergencyNoteModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.ClinicalModels
{
    public class EmergencyNoteModel
    {
        [Key]
        public int EmergencyNoteId { get; set; }
        public int NotesId { get; set; }
        public int PatientId { get; set; }
        public int PatientVisitId { get; set; }
        public string BroughtIn { get; set; }
        public string BroughtBy { get; set; }
        public string Relationship { get; set; }
        public string PhoneNumber { get; set; }
        public string ModeOfArrival { get; set; }
        public string ReferralDoctorOrHospital { get; set; }
        public TimeSpan? TriageTime { get; set; }
        public string TriagedBy { get; set; }
        public bool Trauma { get; set; }
        public string Disposition { get; set; }
        public int? DispositionDepartmentId { get; set; }
        public string ErCourseDescription { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? CreatedOn { get; set; }
        public int? ModifiedBy { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool? IsActive { get; set; }
        public string ICDSelected { get; set; }

    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/clinical-notes/shared/emergency-note.model.ts
import { Time } from "@angular/common";

export class EmergencyNotesModel {
  public EmergencyNoteId: number = 0;
  public NoteId: number = null;
  public PatientId: number = null;
  public PatientVisitId: number = null;
  public BroughtIn: string = null; // Stretcher/ Wheel Chair / Walking
  public BroughtBy: string = null;
  public Relationship: string = null;
  public PhoneNumber: string = null;
  public ModeOfArrival: string = null;
  public ReferralDoctorOrHospital: string = null;
  public TriageTime: Time = null;
  public TriagedBy: string = null; // done by HA or Nurse
  public Trauma: boolean = false;
  public Disposition: string = null;
  public DispositionDepartmentId: number = null;
  public ErCourseDescription: string = null;
  public CreatedBy: number = null;
  public CreatedOn: string = null;
  public ModifiedBy: number = null;
  public ModifiedOn: string = null;
  public IsActive: boolean = true;
  //public SubjectiveNote: SubjectiveNotesModel = new SubjectiveNotesModel();
  //public ObjectiveNote: ObjectiveNotesModel = new ObjectiveNotesModel();
  //public ClinicalDiagnosis: AssessmentAndPlanModel = new AssessmentAndPlanModel();
  //public AllIcdAndOrders: Array<IcdWithOrdersViewModel> = [];
  //public ErCourse: string = null;
}

